### PR TITLE
Create new action

### PR DESCRIPTION
**Pull request by emer11**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2601)

### CODE REVIEW

**Comment by emer11** [comment]
Please review this, @casey-collab

File: english/the-slayer/marshmallow-slayer.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Once upon a time it was the most terrible marshmallow, he slain too many of poor little marshmallows.
+But for fortune it was a heroic marshmallow called Freddie. Lets look at how he saved the little
+marshmallows:
+
+[Freddie](the-slayer/freddie/freddie.md)
```
Comment (casey-collab) [inline english/the-slayer/marshmallow-slayer.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/the-slayer/freddie/freddie.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Freddie saved the little marshmallows stabing the terrible marshmallow with
```
Comment (casey-collab) [inline english/the-slayer/freddie/freddie.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by emer11** [comment]
Please review this, @casey-collab

File: english/the-slayer/marshmallow-slayer.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Once upon a time it was the most terrible marshmallow, he slain too many of poor little marshmallows.
+But for fortune it was a heroic marshmallow called Freddie. Lets look at how he saved the little
+marshmallows:
+
+[Freddie](english/the-slayer/freddie/freddie.md)
```
Comment (casey-collab) [inline english/the-slayer/marshmallow-slayer.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/the-slayer/freddie/freddie.md (hunk 2 above)
Comment (casey-collab) [inline english/the-slayer/freddie/freddie.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by emer11** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :seedling:

**Comment by emer11** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request merged**